### DISCUSSION TITLE
Welcome to dropinambour Discussions!

### DISCUSSION THREAD

**Discussion by gboudreau** [gboudreau/dropinambour#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ghost** [comment]
@gboudreau Hey mate! .... thanks for updating the readme with the Docker steps ... I run UnRAID, which by default has a store called Community Applications. This contains preconfigured Docker templates that members of the community have created, which then allows users to just grab that 'app' and there is minimal config (basically just whatever your specific volume mappings are) to get the container up and running.

I am super green with any php and db stuff, reckon there is any chance you could either add a pre-compiled docker template to the UnRAID community apps (or one of your own repositories) ... or go into a little more depth on how to set up the php and db containers, and how they need to be configured to work with dropinambour?

**Reply by gboudreau** [reply]
You need one DB container, and one dropinambour container. For the latest, the instructions are at the beginning of the installation section of the readme.
For the DB container, you need a MySQL or MariaDB container; if you don't already have one, I'm sure there is an easy to install options in the Unraid store.
Once you have that DB container running, you need to create an empty database in it, and a user that is allowed to access that database (that's step 5 of my instructions; but you'd be better to follow the instructions that are included with the DB app you installed; they are probably easier to follow for your specific situation).
Finally, you need to import the content of `_db/schema.sql` into your empty database, to create empty tables.
For that, you'll need to find the instructions for your DB app on how to execute SQL queries. Once you got that, you can just copy-paste the content of the file into whatever SQL client you were instructed to use. Make sure you select the empty database you created just  before.
Of note : in my instructions, I create a new with this SQL query:

    GRANT ALL on dinb.* TO dinb_user@'localhost' IDENTIFIED BY 'complicated-password';

For you, the `localhost` in that query will need to change. To what : it depends on your DB app, thus why I mentioned you'd be better served by finding instructions about database and user creation in the DB app documentation.

If you are blocked somewhere along the way, let me know.

**Reply by ghost** [reply]
ok no worries, thanks so much - I will give it a shot and see how I go :)
Also just over a week ago, I logged two issues for the XBMCnfoTVImporter:

https://github.com/gboudreau/XBMCnfoTVImporter.bundle/issues/116
https://github.com/gboudreau/XBMCnfoTVImporter.bundle/issues/117

Not sure if you have seen them yet, I tried to detail them as much as I could, so you should have everything in them .... reckon you could take a look at them and let me know what you think?
